### ISSUE TITLE
Shortcuts no longer working 

### ISSUE THREAD

**Issue by mujicapm** [kevinjalbert/todoist-shortcuts-nativefier#10]
Not sure how actively this is being maintained, but shortcuts haven't been working for me at all for several weeks now.

**Comment by kevinjalbert** [comment]
I just updated the submodule (for todoist-shortcuts) to the latest. Try rebuilding now.

If it is still busted, reopen, but it looks like it works for me.

**Comment by mujicapm** [comment]
Back to working. Thanks so much!

**Comment by pat-s** [comment]
Thanks for making this possible! Just checked out this repo and played around a little.

Sadly, many shortcuts are not working for me. Some examples:

- `i`
- `t`: just opens a menu which cannot be triggered
- `shift + c`: toggl extension
- `+`: assign a person

and some more. I can provide more detailed info if you plan to fix them :)

**Comment by kevinjalbert** [comment]
@pat-s so the functionality for the shortcuts are from the https://github.com/mgsloan/todoist-shortcuts repository. I've used git submodules for importing it. I've just updated that now, and so it should have the latest code from that dependency.

I'm curious if you tried the todoist-shortcuts on their own and see if those worked? If so then it might be some odd interaction with that code and nativefier.
